api/server/router/network/network.go

```diff
@@ -6,21 +6,20 @@ import (
 	"github.com/docker/docker/api/server/httputils"
 	"github.com/docker/docker/api/server/router"
 	"github.com/docker/docker/api/server/router/local"
-	"github.com/docker/docker/daemon"
 	"github.com/docker/docker/errors"
 	"golang.org/x/net/context"
 )
 
 // networkRouter is a router to talk with the network controller
 type networkRouter struct {
-	daemon *daemon.Daemon
+	impl   Backend
```

Comment:
I want to change the name to whatever people think makes most sense. I had a distinct lack of imagination here. Look for this change in the next version.